D/Ethernet/EthernetController.cs

```diff
@@ -848,9 +856,12 @@ private void AttachHostEthernet()
             try
             {
 
-                if (Configuration.HostRawEthernetInterfacesAvailable &&
-                    !string.IsNullOrWhiteSpace(Configuration.HostPacketInterfaceName))
-                {
+            	if (Configuration.HostPacketInterfaceName == NethubInterface.NETHUB_NAME) {
+            		_hostInterface = new NethubInterface();
+            		_hostInterface.RegisterReceiveCallback(OnHostPacketReceived);
+            		Console.WriteLine("** connected to nethub");

```

Comment:
Would prefer not to have WriteLines in the code (use Log.Write where applicable.)  See other instances in this and other files as well.